packages/auth/src/shared/baseModel.test.ts

```diff
@@ -17,6 +18,7 @@ export const getPageTests = <Type extends BaseModel>({
       for (let i = 0; i < 22; i++) {
         modelsCreated.push(await createModel())
       }
+      modelsCreated.reverse() // default sort order is DESC
```

Comment:
Small thing, but I think we can optimize the test execution time a bit here by avoiding this `reverse` and the subsequent `reverse` calls in tests where we want ascending.

Like:

```ts
let modelsCreatedAsc: Type[] = []
let modelsCreatedDesc: Type[] = []

beforeEach(async (): Promise<void> => {
  for (let i = 0; i < 22; i++) {
    modelsCreatedAsc.push(await createModel())
    modelsCreatedDesc[21 - i] = await createModel()
  }
})
```

Then use `modelsCreatedAsc` where we use `modelsCreated.reverse()` and `modelsCreatedDesc` where we use `modelsCreated`.

Maybe not a huge factor time-wise since there are 22 only items but it could only help. By my count the `beforeEach` is executing 22 times plus the 7 times where we re-reverse inside tests multiplied by every time `getPage` is called (0 currently? but should probably be used for grants). So we're cutting out 29 o(n) operations (assuming we add it to grants service test).

Side note, I was surprised this works: `modelsCreatedDesc[21 - i] = await createModel()` since the index is out of bounds. I figured it needed an array of length 22. Was a fun little rabbit hole: https://stackoverflow.com/a/22850666/12893676